libs/wallet/src/connectors/injected-connector.ts

```diff
@@ -1,21 +1,85 @@
-import type { VegaConnector } from './vega-connector';
+import { clearConfig, setConfig } from '../storage';
+import type { Transaction, VegaConnector } from './vega-connector';
+
+declare global {
+  interface Vega {
+    getChainId: () => Promise<{ chainID: string }>;
+    connectWallet: (params: { hostname: string }) => Promise<null>;
```

Comment:
I believe the `hostname` param is gone from the desktop api? At least the web wallet does not take any arguments on `connectWallet`